internal/generator/vector/output/elasticsearch/elasticsearch.go

```diff
@@ -43,12 +43,12 @@ id_key = "_id"
 
 func Conf(o logging.OutputSpec, inputs []string, secret *corev1.Secret, op Options) []Element {
```

Comment:
we could use `op Options` to communicate generation only once. but i suspect it will be useful because it is not a pointer

Reply:
we can use `vector.Outputs()` function to generate it only once in the `case logging.OutputTypeElasticsearch:` and use a flag in this function to communicate its generation

Reply:
This has been fixed.